### DISCUSSION TITLE
v7->v8 Migration Guide

### DISCUSSION THREAD

**Discussion by jcburley** [redis/go-redis#2369] [Q&A] [answered]
Is there one? I'm maintaining a package that I didn't write, and I don't know this go-redis package either. Renovate wants us to upgrade to v8, but I'm getting all sorts of compile errors from our code when I try that.

How do I migrate our v7 code to v8 (or, soon apparently, v9)?

**Comment by monkey92t** [comment, marked as answer]
In the v8 version, the ctx parameter is added to all API...You need to change them.
